openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -0,0 +1,306 @@
+{% load i18n %}
+{% load tz %}
+
+<div class="openassessment__student-info staff-info__student__report"
+     data-submission-uuid="{{ submission.uuid }}">
+{% if submission %}
+    <h2 class="title">
+        <span class="label">
+            {% blocktrans with learner=student_username %}
+                Viewing learner: {{ learner }}
+            {% endblocktrans %}
+        </span>
+    </h2>
+
+    <div class="staff-info__status ui-staff__content__section wrapper--ui--collapse staff-info__student__response">
+        <div class="ui-staff ui-toggle-visibility is--collapsed">
+            <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
+                <i class="icon fa fa-caret-right" aria-hidden="true"></i>
+                <span>{% trans "Learner Response" %}</span>
+            </h2>
+            <div class="ui-toggle-visibility__content">
+                {% if workflow_cancellation %}
+                    <p>
+                        {% blocktrans with removed_by_username=workflow_cancellation.cancelled_by removed_datetime=workflow_cancellation.cancelled_at|utc|date:"F j, Y H:i e" %}
+                            Learner submission removed by {{ removed_by_username }} on {{ removed_datetime }}
+                        {% endblocktrans %}
+                    </p>
+                    <!-- Comments: Reason for Cancellation-->
+                    <p>
+                        {% blocktrans with comments=workflow_cancellation.comments %}
+                            Comments: {{ comments }}
+                        {% endblocktrans %}
+                    </p>
+                {% else %}
+                    {% include "openassessmentblock/oa_submission_answer.html" with answer=submission.answer answer_text_label="The learner's response to the question above:" %}
+                {% endif %}
+
+                {% if submission.file_url %}
+                    <a href="{{ submission.file_url }}" class="submission--file">
+                        {% trans "The file associated with this response." %}
+                    </a>
+                    <span>{%  trans "Caution: This file was uploaded by another course learner and has not been verified, screened, approved, reviewed or endorsed by edX. If you decide to access it, you do so at your own risk." %}</span>
+                {% endif %}
```

Comment:
This text comes from Legal so I'm not suggesting edits other than adding a serial comma after "reviewed": "...has not been verified, screened, approved, reviewed, or endorsed by edX."